src/crawler/google_drive/month_review_count_com.google.android.apps.docs_in_en_.csv

```diff
@@ -0,0 +1,37 @@
+time,Frequency
+03/14,2
+09/14,1
+03/15,1
+03/17,1
+09/18,390
```

Comment:
Remove all CSV files in the final check-in code

Reply:
Done